modular_skyrat/master_files/code/modules/research/designs/misc_designs.dm

```diff
@@ -38,3 +38,13 @@
 	build_path = /obj/item/construction/plumbing/research
 	category = list("Tools")
 	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/smartdartgun
+	name = "Medical Smartdart Gun"
```

Comment:
I feel like Smartdarts should be renamed to SmartDart, or something, it'd probably annoy my OCD a little less.

Reply:
Or it could be Smart-Dart. Or just Smart Dart.

Reply:
I could look into this, however I want to be careful with the wording because we have a "Dart Pistol" as a traitor item and I don't want to create confusion.